modules/prebidServerBidAdapter/index.js

```diff
@@ -728,6 +729,7 @@ Object.assign(ORTB2.prototype, {
       if (floor) {
         imp.bidfloor = floor.floor;
         imp.bidfloorcur = floor.currency;
+        bidfloorArray.push(floor.floor);
```

Comment:
yes, you should copy over to floorMin here, and still spare the `Math.min` loop by just keeping the minimum here. (Instead of `bidfloorArray = []` to start, you could set `let floorMin = null` and here do `if (floorMin == null || floorMin < imp.bidfloor) { floorMin = imp.bidfloor }`.

Reply:
@dgirardi thanks, just made those changes to the PR